example/sample_data/sample_table.csv

```diff
@@ -1,6 +1,6 @@
 database,cluster,schema,name,description,tags,is_view,description_source
-hive,gold,test_schema,test_table1,"1st test table","tag1,tag2,pii,high_quality",false,
-dynamo,gold,test_schema,test_table2,"2nd test table","high_quality,recommended",false,
-hive,gold,test_schema,test_view1,"1st test view","tag1",true,
-hive,gold,test_schema,test_table3,"3rd test","needs_documentation",false,
-hive,gold,test_schema,"test's_table4","4th test","needs_documentation",false,
+hive,gold,test_schema,test_table1,1st test table,"tag1,tag2",false,
+dynamo,gold,test_schema,test_table2,2nd test table,recommended,FALSE,
```

Comment:
sorry a bit confused about why these changes were necessary

Reply:
Actually three different reasons (with admittedly varying degree of being necessary 😅):

1. The `pii,high_quality` were originally badges (from when they were a subtype of tags, so before your RFC). Now that badges are a separate thing they suddenly showed up as additional tags. You can cross-reference with the Table Detail Page screenshot on https://www.amundsen.io/amundsen/

2. The quotes were just not needed in this file. So purely done for consistency/cosmetic reasons on that point in this file while I as touching some of the quoted content in 1. anyway.

3. Unrelated (but no functional change): I'm sorry about not mentioning the last ones beforehand with the switching case etc. of `FALSE` and `not_TRUE`. Those are really unrelated, but I came across them when adding unit test code and a fix for this which actually is a separate bug on it's own:

https://github.com/amundsen-io/amundsendatabuilder/blob/453a18b9f70b94226bf01180d7e44b915ca2ddcf/databuilder/extractor/csv_extractor.py#L218-L220 

I will follow up with a separate small PR to fix that bug. The consequence of the bug is that currently all table get the `is_view` in Neo4j and hence displays the `table view` indicator on all datasets on the frontend - not just the. I think there was a recent Slack conversation on it?

Reply:
I see, can we just keep the change explained in 1 as it is related to badges and then address the other issues in a separate PR? I think for 2 is better to keep this the way it was simply because it doesn't make a difference and people have likely been adding the "" to their CSVs, so we would not want to cause any confusion. What do you think?

Reply:
Yes, I can do that, will back it out here and in bring in 3 on the new PR. 

Thanks, and sorry about the confusion.